<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QDialog" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QLabel" name="TEXT_CALIBRATION">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1. Calibration</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CORNER">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1.1 Find Corners</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_INTRINSIC">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1.2 Find Intrinsic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DISTORTION">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1.4 Find Distortion</string>
   </property>
  </widget>
  <widget class="QLabel" name="TEXT_EXTRINSIC">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1.3 Find Extrinsic</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_EXTRINSIC">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>1.3 Find Extrinsic</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BOX_FILE">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CLOSE_ALL_WINDOW">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close All Windows</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
